src/aws-iot-sitewise-mcp-server/awslabs/aws_iot_sitewise_mcp_server/client.py

```diff
@@ -0,0 +1,32 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Centralized AWS IoT SiteWise client creation utility."""
+
+import boto3
+from botocore.config import Config
+
+
+def create_sitewise_client(region: str = 'us-east-1'):
+    """Create a standardized AWS IoT SiteWise client with proper user agent.
+
+    Args:
+        region: AWS region name (default: us-east-1)
+
+    Returns:
+        boto3 IoT SiteWise client instance
+    """
+    config = Config(user_agent_extra='MCP/IoTSiteWiseServer')
```

Comment:
above import the `__version__` from the init...
```suggestion
    config = Config(user_agent_extra=f'AWSLABS/MCP/IoTSiteWiseServer/__version__')
```